### PR TITLE
add example workflow section

### PR DESCRIPTION
**Pull request by boyd-nguyen**
- Added 2 example workflows to About ADO Ecosystem page
- Attached example workflow image (png and svg)

### CODE REVIEW

**Comment by betsybookwyrm** [comment]
If we're going to do both examples together we'll need to wait until the CRT example is approved and finalised - my thought had been to just publish the twitter conversation analysis example today, but if you think it better to do them together then we can wait

**Comment by boyd-nguyen** [comment]
Naomi's given permission to use her topic

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (betsybookwyrm) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:44]: A few notes for the collector script lines:
- spelling of "proprietary" in this and below lines
- instead of proprietary, it might be nicer to say "custom script developed by QUT Digital Observatory to support this project" or something? And maybe link it to the research-project-support resource.
- for the youtube collector maybe add a note that this script is currently being developed into an open source utility

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (betsybookwyrm) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:27]: I think we're going to want to ultimately make the hyperlinks go to resource listings on the ADO website wherever possible, but we don't really have any of these properly listed at the moment! So I'd say this is good for now, and later on we can update the links.

Comment (betsybookwyrm) [reply Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:27]: (this is just a thought for later not a change required now)

**Review by betsybookwyrm — CHANGES_REQUESTED** [review]
Needs some minor wording changes (or at minimum a typo fix) in the second example. Almost there!

**Comment by betsybookwyrm** [comment]
Changes look good, thanks Boyd! I'm just holding off on approving because I think we should have a proper conversation about the external links thing before we merge either PR so that we can make them consistent.

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:23]:
```suggestion
**Twitter conversation analysis**. A researcher is interested in the Twitter conversation around Australia’s federal election. The workflow for answering this problem would include a preliminary feasibility check (to ensure there are sufficient data for analysis), followed by data collection, processing, and analysis.
```

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:28]:
```suggestion
<li><a href="https://www.ado.eresearch.unimelb.edu.au/tutorials/" target="_blank">ADOReD</a>: high-level social media analysis via an interactive dashboard. It can be used to derive a list of tweet IDs relevant to the research question.</li>
```

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:30]:
```suggestion
<li><a href="https://pypi.org/project/tidy-tweet/" target="_blank">tidy_tweet</a>: in-development open-source Python library for processing raw responses from Twitter API into an SQLite database.</li>
```

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:31]:
```suggestion
<li>tweet_exploR: in-development R package providing descriptive statistics and visualisation of Twitter data within an SQLite database produced by tidy_tweet.</li>
```

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:40]: Maybe we should have a new line after the bold heading to make it clear that it's a different example?

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:44]:
```suggestion
<li><a href="https://pypi.org/project/tidy-tweet/" target="_blank">tidy_tweet</a>: in-development open-source Python library for processing raw responses from Twitter API into an SQLite database.</li>
```

**Review by Quiet27 — CHANGES_REQUESTED** [review]
Looks great!

I have one other suggestion: Remove the title from the example workflow chart, since it's contained within a section that already explains what it is. If you'd prefer to keep it, can we shorten it to "Analysis of Twitter conversation around Australia's Federal Election".

**Comment by boyd-nguyen** [comment]
Thanks for your feedback Alice and Betsy! I've edited the text and image accordingly. 
Another thing I did was remove the h4 "Contents" heading before the table of contents to avoid skipping heading levels. Having a look at the page I think the ToC should be clear enough without that heading. But let me know if you think otherwise.

**Review by Quiet27 — APPROVED** [review]
Nothing further from me, I think it's good to go! Thanks Boyd 😸

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr
Comment (betsybookwyrm) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:26]: Looks great, Boyd - there's just a git diff artefact here and line 30 below. Once that's cleaned up, just go ahead and merge it :)

**Review by betsybookwyrm — DISMISSED** [review]

**Review by betsybookwyrm — APPROVED** [review]
Thanks, Boyd!

**Pull request merged**
